old-backup/ingest-method-rewrites.md

```diff
@@ -0,0 +1,48 @@
+<!--DO NOT REVIEW-->
```

Comment:
Like it says on the tin, this content is for the next PR.
It was written during work on this PR whilst conceptualising information.